### PR TITLE
[rpm] Integrate 1.61.1. JB#50106

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Review by mkosola — APPROVED** [review]

File: rpm/c-ares.spec [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+%prep
+%autosetup -p1 -n %{name}-%{version}/%{name}
+
+f=CHANGES ; iconv -f iso-8859-1 -t utf-8 $f -o $f.utf8 ; mv $f.utf8 $f
+
+%build
+autoreconf --force --install
+%configure --enable-shared --disable-static \
+           --disable-dependency-tracking
+%{__make} %{?_smp_mflags}
```
Comment (okodron) [inline rpm/c-ares.spec]: `%make_build` could be used.

File: rpm/c-ares.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+%install
+rm -rf $RPM_BUILD_ROOT
+make DESTDIR=$RPM_BUILD_ROOT install
```
Comment (okodron) [inline rpm/c-ares.spec]: Just `%make_install`.

File: rpm/c-ares.spec [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+%description
+c-ares is a C library that performs DNS requests and name resolves
+asynchronously. c-ares is a fork of the library named 'ares', written
+by Greg Hudson at MIT.
+
+%package devel
+Summary: Development files for c-ares
+Requires: %{name} = %{version}-%{release}
+Requires: pkgconfig
```
Comment (okodron) [inline rpm/c-ares.spec:21]: Not sure that it's necessary here.

File: rpm/c-ares.spec (hunk 1 above)
Comment (rainemak) [reply rpm/c-ares.spec]: Yeap. will update. Good catch. Btw, baseline is from fedora's spec.

**Pull request merged**